### PR TITLE
better 

### PR DESCRIPTION
**Pull request by kaif-00z**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -22,7 +22,7 @@ RUN ver=$(curl https://raw.githubusercontent.com/BLUE-DEVIL1134/UltroidCli/main/
 RUN chmod u+x ultroid
 
 # Clone the repository and install the dependencies
-RUN ./ultroid init
+RUN . /ultroid init
```
Comment (iAkashPattnaik) [inline Dockerfile]: Son, it's a Linux command, `./filename` is to execute it.

File: app.json [hunk 2]
```diff
@@ -46,6 +46,16 @@
             "description": "Name of your Heroku app, given in the first blank on this page.",
             "value": "",
             "required": true
+        },
+        "LOG_CHANNEL": {
```
Comment (iAkashPattnaik) [inline app.json:50]: Dude, don't you dare..
Ultroid auto creates these stuff...

Comment (kaif-00z) [reply app.json:50]: if anyone not wanted to auto create these stufff so,

Comment (iAkashPattnaik) [reply app.json:50]: Then they can do it by deploying, then adding these stuff to the config vars and restart.
It's best to keep this repo as minimal as possible.

Comment (kaif-00z) [reply app.json:50]: ok

**Pull request closed without merging**